website/docs/r/org_user.html.markdown

```diff
@@ -0,0 +1,165 @@
+  on read. It is inspected on create and modify. On modify, the absence of this element indicates that the password 
+  should not be changed.
+* `password_file` (Optional, but required if `password` was not given). A text file containing the password. 
+  Using this property instead of `password` has the advantage that the sensitive data is not saved into Terraform state 
+  file. The disadvantage is that a password change requires also changing the file name.
+* `provider_type` - (Optional) Identity provider type for this this user. One of: `INTEGRATED`, `SAML`, `OAUTH`. The default
+   is `INTEGRATED`.
+* `role` - (Required) The role of the user. Role names can be retrieved from the organization. Both built-in roles and
+  custom built can be used. The roles normally available are:
+    * `Organization Administrator`
+    * `Catalog Author`
+    * `vApp Author`
+    * `vApp User`
+    * `Console Access Only`
+    * `Defer to Identity Provider`
+* `full_name` - (Optional) The full name of the user.
+* `description` - (Optional) An optional description of the user.
+* `telephone` - (Optional) The Org User telephone number.
+* `email_address` - (Optional) The Org User email address. Needs to be a properly formatted email address.
+* `instant_messaging` - (Optional) The Org User instant messaging.
+* `is_enabled` - (Optional) True if the user is enabled and can log in. The default is `false`.
+* `is_group_role` - (Optional) True if this user has a group role.. The default is `false`.
+* `is_locked` - (Optional) True if the user account has been locked due to too many invalid login attempts. A locked 
+  user account can be re-enabled by updating the user with this flag set to false. Only the system can set the value to 
+  true. 
+* `take_ownership` - (Optional) Take ownership of user's objects on deletion.
+* `deployed_vm_quota` - (Optional) Quota of vApps that this user can deploy. A value of 0 specifies an unlimited quota.
+  The default is 10.
+* `stored_vm_quota` - (Optional) Quota of vApps that this user can store. A value of 0 specifies an unlimited quota.
+  The default is 10.
+
+
+## Attribute Reference
+
+The following attributes are exported on this resource:
+
+* `id` - The ID of the Organization user
+
+
+## Importing
+
+~> **Note:** The current implementation of Terraform import can only import resources into the state. It does not generate
+configuration. [More information.][docs-import]
+
+An existing user can be [imported][docs-import] into this resource via supplying the full dot separated path for an
+org user. For example, using this structure, representing an existing user that was **not** created using Terraform:
+
+```hcl
+resource "vcd_org_user" "my-org-admin" {
+  org  = "my-org"
+  name = "my-org-admin"
+  role = "Organization Administrator"
+}
+```
+
+You can import such user into terraform state using this command
+
+```
+terraform import vcd_org_user.my-org-admin my-org.my-org-admin
```

Comment:
I am kind'a lost with this example. @lvirbalas  @Didainius  Are understanding this?

Reply:
I do, but I can be a bad example as I've played with this too much :)

Reply:
@vbauzysvmware Please read the docs. Here's the executive summary:

Terraform import uses a combination of a **minimal definition** of a resource + a import command to retrieve data from the provider and add it to the state file.

The syntax is `terraform import resource_type.resource_name resource_path`
where `resource_path` is how we get to the resource from the top. In our case is `org_name.user_name`.
In places more complicated it could be `org_name.vdc_name.edge_name.service_name`

Reply:
Thank you!